lib/cisco_node_utils/cmd_ref/bridge_domain_vni.yaml

```diff
@@ -0,0 +1,34 @@
+# bridge_domain
+# bridge_domain feature is available only on n7k
+---
+_exclude: [N3k, N5k, N6k, N8k, N9k, ios_xr]
+
+create:
+  set_value: "bridge-domain <bd>"
+
+member_vni:
+  get_command: "show running-config bridge-domain | inc 'member vni'"
+  get_value: '/^  member vni (.*)/'
+  set_value: "bridge-domain <bd> ; <membstate> member vni <membvni>"
+  default_value: ''
```

Comment:
In this case the running-config output is a bit useless thanks to the multiple bridge-domain entries, i.e. this is less than helpful:

```
bridge-domain 100

bridge-domain 100 
bridge-domain 100
  member vni 5000
```

So... I think we have no choice in this case but to use `show bridge-domain <bd>:

```

n7k8(config)# sh bridge-domain 100

Bridge-domain 100  (0 ports in all)
Name:: Bridge-Domain100
 Administrative State: UP               Operational State: DOWN
        vni5000
```

```
member_vni:
  get_command: "show bridge-domain"
  context: 'Name:: Bridge-Domain<bd>'
  get_value: '/vni(\S+)/'
```